src/components/Setup/index.tsx

```diff
@@ -32,15 +34,19 @@ const Setup: React.FC = () => {
   const [currentStep, setCurrentStep] = useState(1);
   const [plexSettingsComplete, setPlexSettingsComplete] = useState(false);
   const router = useRouter();
+  const { locale } = useLocale();
 
   const finishSetup = async () => {
-    setIsUpdating(false);
+    setIsUpdating(true);
     const response = await axios.post<{ initialized: boolean }>(
       '/api/v1/settings/initialize'
     );
 
     setIsUpdating(false);
     if (response.data.initialized) {
+      await axios.post('/api/v1/settings/main', { locale });
```

Comment:
Should we really be changing the global display language here or just the users?

Reply:
I think that the server owner's family/friends would be likely to have similar language preferences, so I think that the locale selected during setup is a pretty safe default setting.

Reply:
I think that always setting the server default display language to English would be less ideal than using the user-selected setup language (in that it probably isn't what the user would want, and would force them to find and change the setting manually).